staker-rewards/README.adoc

```diff
@@ -33,9 +33,11 @@ For each Operator:
 * Calculate the sum of ETH withdrawn for the entire interval as `ETH_withdrawn`.
   
 * Calculate the total ETH under management as 
-  `ETH_locked = ETH_bonded + ETH_unbonded - ETH_withdrawn`.
-  In case the operator is undelegating or has been de-authorized for the
-  sorition pool, `ETH_locked = min(ETH_bonded_T_start, ETH_bonded_T_end)`.
+  `ETH_locked = ETH_bonded + ETH_unbonded - ETH_withdrawn`. In case the operator
+  is undelegating OR
+  has been de-authorized for the sorition pool in current interval OR
+  is not authorized for the sortition pool at the interval start
+  their `ETH_locked = min(ETH_bonded_T_start, ETH_bonded_T_end)`.
```

Comment:
We also need to update requirements below:
- If the operator does not have BondedECDSAKeepFactory contract or tBTC sortition pool authorized at T_start, they are not getting any rewards for interval T.
 - If the operator deauthorized tBTC sortition pool between T_start and T_end, they are not getting any rewards for interval T.

into:

- If the operator does not have BondedECDSAKeepFactory contract authorized at T_start, they are not getting any rewards for interval T
- if the operator does not have tBTC sortition pool authorized at T_start and ETH_bonded = 0 for that operator, they are not getting any rewards for interval T.

Reply:
https://github.com/keep-network/keep-ecdsa/pull/828/commits/0db6c14858f3d2e9bf2be5068b6c84df4be138cf